modules/ROOT/pages/streaming-about.adoc

```diff
@@ -271,6 +248,17 @@ increments at 100 objects per increment and allows a maximum buffer size of
  </sfdc:query>
 ----
 
+[[disable_repeatable_streaming]]
+== Disabling Repeatable Streaming
+
+You can disable repeatable streaming through the `non-repeatable-stream` and `non-repeatable-iterable` strategies. The strategy to use depends on the type of stream.
+
+Use this option only if you are certain that there is no need to consume the stream several times and only if you need a very tight optimization for performance and resource consumption. 
+
+When making that decision, consider the following: 
```

Comment:
This seems a little out of place, or repetitive of the two items just listed: frequency of consumption and very strict optimization.
Do the following texts belong under one of the two items I am suggesting that we list, as further info on each?

Reply:
If you keep the texts where they are, do we really need this intro sentence?
```suggestion
```